modules/core/src/lib/constants.ts

```diff
@@ -112,5 +112,6 @@ export const EVENTS = {
  */
 export const OPERATION = {
   DRAW: 'draw',
-  MASK: 'mask'
+  MASK: 'mask',
+  TERRAIN: 'terrain'
```

Comment:
Good time to start providing modern alternatives to enums (better typed, and don't require app to import the enum)
```suggestion
/** Rendering operations on layers */
export type Operation = 'draw' | 'mask' | 'terrain';

/** @deprecated Use string constants directly */
export const OPERATION = {
  DRAW: 'draw',
  MASK: 'mask',
  TERRAIN: 'terrain'
}
```